dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:3 as base
+
+WORKDIR /opt/todo_app
+RUN curl -sSL https://install.python-poetry.org | python -
+
+ENV PATH=/root/.local/bin:$PATH
+
+COPY pyproject.toml .
+COPY poetry.lock .
+
+RUN poetry install
+
+COPY /todo_app ./todo_app
+
+FROM base as production
+
+EXPOSE 5000
```

Comment:
This ought to be `EXPOSE 8000` to properly document which port this image listens to.